tests/performance/Taskfile.yaml

```diff
@@ -29,14 +31,33 @@ tasks:
   apply:all:
     desc: "Apply disks and virtual machines in two steps (in two different releases)."
     cmds:
-      - task apply:disks
-      - task apply:vms
+      - task: apply:disks
+        vars:
+          COUNT: "{{ .COUNT }}"
+          NAMESPACE: "{{ .NAMESPACE }}"
+          STORAGE_CLASS: "{{ .STORAGE_CLASS }}"
+          NAME_PREFIX: "{{ .NAME_PREFIX }}"
+          RESOURCES_PREFIX: "{{ .RESOURCES_PREFIX }}"
+      - task: apply:vms
+        vars:
+          COUNT: "{{ .COUNT }}"
+          NAMESPACE: "{{ .NAMESPACE }}"
+          NAME_PREFIX: "{{ .NAME_PREFIX }}"
+          RESOURCES_PREFIX: "{{ .RESOURCES_PREFIX }}"
```

Comment:
why duplicate variables if they are already globally declared?

Reply:
When passing variables to the task (COUNT=5 task apply:all), in subtasks, the values of global variables remain unchanged; it's necessary to explicitly pass new values ​​for the variables.

Reply:
:okay: